@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -290,4 +296,15 @@ export class OmvDataSource extends TileDataSource<OmvTile> {
             storageLevelOffset: this.storageLevelOffset
         });
     }
+
+    setEnableElevationOverlay(enable: boolean) {
+        if (this.m_decoderOptions.enableElevationOverlay !== enable) {
+            this.m_tileLoaderCache.clear();
```

Comment:
part of this code seems similar to that in setStyleSet, would it make sense to create a method to bundle this together, something like `reconfigureDecoders` (for want of a better name).

Reply:
done.